lib/osproject_android.rb

```diff
@@ -50,8 +50,8 @@ def add_sdk!
     rescue
       puts "File not found: " + build_gradle_dir 
       puts "Call CLI tool from base project directory"
-      error_track_message = "error=user called CLI from invalid directory file: " + build_gradle_dir + " not found;"
-      network_handler.send_track_from_message(os_app_id, 'android', lang, success_mesage, error_track_message)
+      
+      network_handler.send_track_from_error(os_app_id, 'android', lang, success_mesage, "user called CLI from invalid directory file: " + build_gradle_dir + " not found;")
```

Comment:
I think I would have left this one as you had it before, with the `error_track_message` definition as  param.